src/backend/middleware/index.ts

```diff
@@ -21,8 +21,10 @@ function secureHeaders(router: Router, frontendApp: Express) {
 				"'unsafe-inline'", "'strict-dynamic'", (_: Request, res: Response): string => `'nonce-${res.locals.nonce}'`],
 			// Allow XMLHttpRequest/fetch requests
 			connectSrc: ["'self'", process.env.APP_URL],
-			// Allow <style> tags hosted by ourselves as well as style="" attributes
-			styleSrc: ["'self'", "'unsafe-inline'"],
+			// Allow <style> tags hosted by ourselves and cdnjs, as well as style="" attributes
+			styleSrc: ["'self'", "'unsafe-inline'", "https://cdnjs.cloudflare.com"],
```

Comment:
I wonder what do we need it for? 
I assume it somehow relates to Atlastunnel, I wonder what will be broken if we won't put it here?